src/ChairmarksExtras.jl

```diff
@@ -12,7 +12,7 @@ Benchmark `f` and return the fasted runtime in seconds.
 
 Equivalent to `(@b args...).time`.
 
-See `@be` for more info.
```

Comment:
The point of "for more info" is to clarify that `@b` gives information on what `args...` are. It's not just a "see also" to a related function it's that this function is defined fairly transparently in terms of `@be`.

I like the cross-link (though perhaps that will break docs building because `@be` is not defined in this repo, which would explain why I omitted the cross-link) but I do want to keep some indication that this is more than a "see also".

Reply:
How about now?

Reply:
I don't see how adding `Chairmarks.` addresses this? I still prefer the "for more info" wording over "see also"

Reply:
I refer directly to `@be` because that is where the arguments are documented. I don't include a "see also" for `@b` because anyone reading the primary docstring knows that `@belapsed` is degined in terms of `@b` and knows to look there for more info already.

Reply:
So you would prefer just "See [`@be`](@ref)` or [`@b`](@ref) for more info"?